src/ui/starter-select-ui-handler.ts

```diff
@@ -1601,87 +1605,94 @@ export default class StarterSelectUiHandler extends MessageUiHandler {
               },
             },
           );
+
+          // Manage Moves options
           if (this.speciesStarterMoves.length > 1) {
-            // this lets you change the pokemon moves
-            const showSwapOptions = (moveset: StarterMoveset) => {
+            const getMoveOptions = (
+              moves: MoveId[],
+              selectHandler: (moveId: MoveId, index: number, currentMoveId?: MoveId, currentIndex?: number) => boolean,
+              cancelHandler: () => boolean,
+              currentMoveId?: MoveId,
+              currentIndex?: number,
+            ): OptionSelectModeConfig => {
+              const options: OptionSelectItem[] = moves.map((moveId: MoveId, index: number): OptionSelectItem => {
+                return {
+                  label: allMoves[moveId].name,
+                  handler: () => selectHandler(moveId, index, currentMoveId, currentIndex),
```

Comment:
```suggestion
                  handler: () => return selectHandler(moveId, index, currentMoveId, currentIndex),
```
Think I forgot that. Will double check

Reply:
you dont need `return` in this case

Reply:
The `return` is implied because there's no `{}`.